app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -444,7 +443,7 @@ class ExplorationActivityTest {
   }
 
   @Test
-  fun testExploration_overflowMenu_isDisplayedSuccessfully() {
+  fun testExploration_openBottomSheet() {
```

Comment:
What's the expected outcome? The action seems to be opening the bottom sheet, but it's not clear what this test is making sure works.

Perhaps instead:

``testActivity_initialState_openBottomSheet_showsBottomSheet``

Might help clarify?